decidim-comments/app/frontend/comments/add_comment_form.component.jsx

```diff
@@ -228,6 +229,7 @@ const AddCommentFormWithMutation = graphql(gql`
             if (comment.id === commentableId) {
               return {
                 ...comment,
+                hasReplies: true,
```

Comment:
Let me answer this:

- I added the flag because the `replies` field is not included for the last level of replies. Remember a reply is still a comment so it uses the same graphql fragment. I used the condition `replies && replies.length > 0` in the past but it felt very weak too me and the component knew too much. The flag `hasReplies` comes from the backend so we can control the logic from the API.
- The flag is set to true whenever a new comment is added and it is a reply. Since the `addComment` mutation is updating the `GetComments` query we need to be sure it's updated correctly. If it's not updated, it causes an annoying flickering of the state since the `GetComments` query is refetched every 10 seconds.